### ISSUE TITLE
Impossible to build blog with "custom" _tepmplateFiles.cshtml

### ISSUE THREAD

**Issue by tesar-tech** [statiqdev/Statiq#746]
I created blog with: `wyam new -r blog`, and build it with `wyam -p -w` and it worked...
I want to customize template file, so I copied everything from `C:\Users\me\.nuget\packages\Wyam.Blog.CleanBlog.1.7.2\content` to my `input` folder and it crashes after build in `RenderBlogPosts` pipeline with error:

```
 Executing pipeline "RenderBlogPosts" (8/15) with 5 child module(s)
            Exception while processing document file:///C:/Users/me/Desktop/newblog/input/posts/first-post.md in module Razor: The partial view '_PostFooter.cshtml' was not found. The following locations were searched:
/input/_PostFooter.cshtml
            Error while executing module Razor
        Error while executing pipeline RenderBlogPosts
Exception during execution: System.AggregateException: One or more errors occurred. ---> System.InvalidOperationException: The partial view '_PostFooter.cshtml' was not found. The following locations were searched:
/input/_PostFooter.cshtml
```

I am pretty sure there is `_PostFooter.cshtml` file. 

What is wrong here?
Thanks.

**Comment by daveaglick** [comment]
You hit a bug that's been fixed here: https://github.com/Wyamio/Wyam/commit/1747b1b40001ae73afdfaa9120ec8abfcef980c2

I'll see about backporting the fix to the 1.7.x versions - keep an eye out for 1.7.3.

**Comment by daveaglick** [comment]
1.7.3 is rolling out now which should resolve this issue for you (it'll take a few minutes for NuGet to propagate). Please feel free to comment/reopen if you're still having trouble.
